internal/plugins/openshift/v1/init.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	// The current OCP release version.
-	ocpProductVersion = "4.15"
+	ocpProductVersion = "4.17"
```

Comment:
Apart from this change, I hope everything else is exact same as https://github.com/openshift/ocp-release-operator-sdk/pull/387

Reply:
Yes. That's correct.